content/docs/common-areas/cdn

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
The file is missing an extension. I suppose that is why the build failed (but please check anyway).